### ISSUE TITLE
Accessibility - Unable to navigate using arrow keys in single-selection-datagrid in Firefox/Edge/IE11

### ISSUE THREAD

**Issue by pavandixit93** [vmware/clarity#5237]
## Describe the bug
While using `single-selection-datagrid` I found that `name` attribute is not set and missing on `<input type="radio".../>` which is rendered under `<clr-radio-wrapper.../>` and due to which I am not able to navigate radio selection using arrow keys in Firefox, Edge and IE11. If I add random `name` attribute on each radio elements I can easily navigate using arrow keys.
This also violates radio selection keyboard navigation [wai-aria-practices - Radio component](https://www.w3.org/TR/2017/WD-wai-aria-practices-1.1-20170628/examples/radio/radio-1/radio-1.html)

## How to reproduce
_Please provide a link to a reproduction scenario using one of the [Clarity Stackblitz templates](https://stackblitz.com/@clr-team). Reports without a clear reproduction may not be prioritized until one is provided._

1.  Please visit [https://clarity.design/documentation/datagrid/selection-single](https://clarity.design/documentation/datagrid/selection-single) in Firefox or Edge or IE11
2.  Click on any datagrid row and it will select the datagrid row.
3.  Press the `down arrow key` on keyboard 
4.  It should select or highlight next datagrid row 
5. This works fine on Chrome but it fails on Firefox/Edge/IE11.

## Expected behavior
I should be able to navigate using arrow key in [https://clarity.design/documentation/datagrid/selection-single](https://clarity.design/documentation/datagrid/selection-single)
_A clear and concise description of what you expected to happen._

## Versions

**App**

- Angular: [e.g. 6]
- Node: [e.g. 8.10.0]
- Clarity: [e.g. 0.12.5]

**Device:**

- Type: [e.g. MacBook]

## Additional notes

_Add any other notes about the problem here._

**Comment by bdryanovski** [comment]
@pavandixit93 thank you for reporting this issue. 

There is an open [issue](https://github.com/vmware/clarity/issues/5136) about that already. So I'm closing this in its favor.
